src/cosalib/aws.py

```diff
@@ -50,7 +50,8 @@ def aws_run_ore_replicate(build, args):
 
     region_list = list(set(args.region) - set(duplicates))
     if len(region_list) == 0:
-        raise Exception("no new regions detected")
+        print("no new regions detected")
+        sys.exit(0)
```

Comment:
I think you need to import the `sys` package(https://jenkins-coreos-ci.apps.ci.centos.org/blue/organizations/jenkins/github-ci%2Fcoreos%2Fcoreos-assembler/detail/PR-1570/1/pipeline)

Reply:
Thanks, fixed in latest push!